neuraxle/steps/flow.py

```diff
@@ -0,0 +1,199 @@
+from abc import abstractmethod
+from collections import OrderedDict
+from typing import Union
+
+from py._log import warning
+
+from neuraxle.base import BaseStep, MetaStepMixin, DataContainer, ExecutionContext
+from neuraxle.hyperparams.space import HyperparameterSamples
+from neuraxle.pipeline import Pipeline
+
+
+class ForceHandleMixin:
+    """
+    A pipeline step that only requires the implementation of handler methods :
```

Comment:
Add something like
> ... but that forbids only implementing `fit` or `transform` or `fit_transform` without the handles. So it forces the handles.

Reply:
This is not resolved. I think the error is here. You say `... that *only* requires the implementation of  ...` in the docstring. But this is wrong. It should instead say `... that requires the implementation *only* of  ...`. The difference of phrasing is big. The error was also here in the docstring of a new example (which I fixed myself in the branch).